development/compiling/compiling_for_web.rst

```diff
@@ -10,21 +10,31 @@ Requirements
 
 To compile export templates for the Web, the following is required:
 
--  `Emscripten SDK <http://emscripten.org/>`__ (Install in a path without
-   spaces, i.e. not on "Program Files")
--  `Python 2.7+ <https://www.python.org/>`__ (3.0 is
-   untested as of now)
+-  `Emscripten <http://emscripten.org/>`__, plus `LLVM <https://llvm.org/>`__ and `Binaryen <https://github.com/WebAssembly/binaryen>`__ if you want to compile for *Webassembly* (Godot 3). The easiest way to install them if they are not up to date in your Linux package manager is to use the `Emscripten SDK <http://emscripten.org/>`__ (Install in a path without spaces, i.e. not on "Program Files")
+-  `Python 2.7+ <https://www.python.org/>`__ (3.0 is untested as of now)
 -  `SCons <http://www.scons.org>`__ build system
 
 Building export templates
 -------------------------
 
 Start a terminal and set the environment variable ``EMSCRIPTEN_ROOT`` to the
-installation directory of Emscripten::
+installation directory of Emscripten:
+
+-  If you installed Emscripten via the Emscripten SDK, you will declare the variable with a path to your downloaded folder:
```

Comment:
This colon needs to be a double colon, so the following paragraph is displayed with a fixed-width typeface.